integration/src/integration/java/pl/allegro/tech/hermes/integration/helper/graphite/GraphiteMockServer.java

```diff
@@ -68,7 +68,7 @@ public void expectMetric(String metricNamePattern, double value) {
     }
 
     public void waitUntilReceived() {
-        await().atMost(Duration.TEN_SECONDS).until(() -> {
+        await().atMost(Duration.ONE_MINUTE).until(() -> {
```

Comment:
why do we default to one minute everywhere? just get higher multiplier for travis, but don't make me wait one minute for test to fail (x 100) while i'm developing